README.md

```diff
@@ -19,3 +19,51 @@ To run this application, you need to first clone or download the ‘create a fil
+
+### Localization
+The text of a context menu list and the tooltips being displayed in the user interface can be localized to any supported language. The Blazor File Manager component uses a localization library (l10) to localize the text values.
+
+### Adapts to any resolution 
+The File Manager is designed to be responsive and adaptable, with a layout that works well on devices including desktops, touch screens, and smart phones.
+
+### Searching
+The Blazor File Manager control contains built-in search functionality that allows you to search for and access a file or folder in the current path and its child folders.
+
+### Multiple-file selection
+The Blazor File Manager component has built-in support for the selection of multiple files or folders. [Multiple-file selection](https://blazor.syncfusion.com/documentation/file-manager/multiple-file-selection?utm_source=github&utm_medium=listing&utm_campaign=blazor-file-manager-github-samples) will be effective when a single operation like download or delete is performed over multiple files in the file system.
+
+### Drag and Drop 
+The file manager allows files and folders to be moved within the file system by [drag and dropping](https://blazor.syncfusion.com/documentation/file-manager/drag-and-drop?utm_source=github&utm_medium=listing&utm_campaign=blazor-file-manager-github-samples) them.
+
+
+## Not sure how to create your first Blazor File Manager? Our documentation can help.
+* [Getting Started with Blazor Server-Side File Manager](https://blazor.syncfusion.com/documentation/file-manager/getting-started?utm_source=github&utm_medium=listing&utm_campaign=blazor-file-manager-github-samples)
+
+## Related links
+[Learn More about Blazor File Manager](https://www.syncfusion.com/blazor-components/blazor-file-manager?utm_source=github&utm_medium=listing&utm_campaign=blazor-file-manager-github-samples) <br/><br/>
+[Download Free Trial](https://www.syncfusion.com/downloads?utm_source=github&utm_medium=listing&utm_campaign=blazor-file-manager-github-samples) <br/><br/>
+[Pricing](https://www.syncfusion.com/sales/teamlicense?utm_source=github&utm_medium=listing&utm_campaign=blazor-file-manager-github-samples) <br/><br/>
+[Documentation](https://blazor.syncfusion.com/documentation/file-manager/getting-started?utm_source=github&utm_medium=listing&utm_campaign=blazor-file-manager-github-samples) <br/><br/>
+[Online Examples](https://blazor.syncfusion.com/demos/file-manager/overview?theme=bootstrap5?utm_source=github&utm_medium=listing&utm_campaign=blazor-file-manager-github-samples) <br/><br/>
+[Community Forums](https://www.syncfusion.com/forums/blazor-components?control=filemanager?utm_source=github&utm_medium=listing&utm_campaign=blazor-file-manager-github-samples) <br/><br/>
+[Suggest a feature](https://www.syncfusion.com/feedback/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-file-manager-github-samples)
```

Comment:
All this should be in oreder
[Learn More about .NET MAUI Chips]

[Download Free Trial]

[Pricing]

[Documentation]

[View Demos]

[Community Forums]

[Suggest a feature or report a bug]

[Online example]

[Knowledge base]